app/sprites.py

```diff
@@ -323,5 +340,239 @@ def generate_position() -> tuple:
         return width, height, direction
 
     def kill(self) -> None:
+        self.some_effect()
         super().kill()
         self.engine.score += self.score_points
+
+    def some_effect(self):
+        effect = random.choice(["", "", "", "healing", "fast", "slow"])
```

Comment:
It looks a little bit weird. Can you implement this without empty strings and hardcoded names of effects.